benchmarking/weight_file_extrinsic_template.go

```diff
@@ -0,0 +1,19 @@
+// THIS FILE WAS GENERATED USING GOSEMBLE BENCHMARKING PACKAGE
+package benchmarking
+
+import (
+	sc "github.com/LimeChain/goscale"
+	primitives "github.com/LimeChain/gosemble/primitives/types"
+)
+
+func extrinsicWeightFn(dbWeight primitives.RuntimeDbWeight) primitives.Weight {
```

Comment:
Add a comment to note that this is used as a template and any changes made here (the compiler won't complain) could accidentally break the automatic generation of weights later.